.github/workflows/code_checks.yml

```diff
@@ -134,11 +135,7 @@ jobs:
         id: test-status
         if: always()
         run: |
-          if [ "${{ job.status }}" = "success" ]; then
-            echo "status=success" >> $GITHUB_OUTPUT
-          else
-            echo "status=failure" >> $GITHUB_OUTPUT
-          fi
+          echo "status=${{ job.status }}" >> $GITHUB_OUTPUT
 
```

Comment:
Simplified logic due to job.status is already defined. No reason to check if success or not.